pom.xml

```diff
@@ -145,6 +145,11 @@
             <artifactId>validation-api</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.google.code.findbugs</groupId>
+            <artifactId>annotations</artifactId>
+            <version>3.0.0</version>
```

Comment:
@cvrebert can we find some other workaround? I really don't like the idea of adding this dependency to the "compile" scope